core/webgl-compatibility/src/test/utils/webpack.config.js

```diff
@@ -49,7 +49,7 @@ function createConfig(shouldInstrument) {
       new webpack.DefinePlugin({
         "process.env": Object.keys(process.env)
           .reduce((env, key) => {
-            env[key] = JSON.stringify(process.env[key]);
+            env[key] = JSON.stringify({ [key]: process.env[key] });
```

Comment:
pretty sure this is wrong in every webpack config file

Reply:
changed for all instances

Reply:
im not sure if this is right.
i imagine its this change which is causing the integration tests to fail? https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=2150186&view=logs&j=e8109472-6daa-581c-ccd8-5247bc21e57f&t=d406a847-95c2-54af-05cd-cda164705e7d&l=113

Reply:
undid change, misread comment, thought you meant to make everyting with { [key]: process.env[key] }